### PR TITLE
Final changes for v1.2.1

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Version 1.2.1

### CODE REVIEW

File: app/src/main/java/org/openobservatory/ooniprobe/activity/RunTestActivity.java [hunk 1]
```diff
@@ -66,9 +66,10 @@ protected void onCreate(Bundle savedInstanceState) {
         if (Intent.ACTION_VIEW.equals(intent.getAction())) {
             Uri uri = intent.getData();
             String mv = uri.getQueryParameter("mv");
-            //TODO cut all the stuff after the BuildConfig.VERSION_NAME like -rc.1
+            String[] split = BuildConfig.VERSION_NAME.split("-");
+            String version_name = split[0];
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/activity/RunTestActivity.java]: What happens if `version_name` does not contain a "-"?

Comment (lorenzoPrimi) [reply app/src/main/java/org/openobservatory/ooniprobe/activity/RunTestActivity.java]: Already tested, you still have a 0 element in the array, the full string

Comment (bassosimone) [reply app/src/main/java/org/openobservatory/ooniprobe/activity/RunTestActivity.java]: Most excellent!

**Comment by lorenzoPrimi** [comment]
Uploaded to Google Play

**Pull request merged**